src/sentry/south_migrations/0386_auto__add_field_groupassignee_team__chg_field_groupassignee_user.py

```diff
@@ -0,0 +1,1078 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```

Comment:
I am 100% doing this by hand since the SQL that South spits out is not great at all.

```
 > sentry:0386_auto__add_field_groupassignee_team__chg_field_groupassignee_user
   = ALTER TABLE "sentry_groupasignee" ADD COLUMN "team_id" bigint NULL; []
   = ALTER TABLE "sentry_groupasignee" ALTER COLUMN "team_id" TYPE bigint, ALTER COLUMN "team_id" DROP NOT NULL, ALTER COLUMN "team_id" DROP DEFAULT; []
 [...]
   = ALTER TABLE "sentry_groupasignee" DROP CONSTRAINT "user_id_refs_id_f4dcb8d1" []
   = ALTER TABLE "sentry_groupasignee" ALTER COLUMN "user_id" TYPE integer, ALTER COLUMN "user_id" DROP NOT NULL, ALTER COLUMN "user_id" DROP DEFAULT; []
   = ALTER TABLE "sentry_groupasignee" ADD CONSTRAINT "user_id_refs_id_f4dcb8d1" FOREIGN KEY ("user_id") REFERENCES "auth_user" ("id") DEFERRABLE INITIALLY DEFERRED; []
   = ALTER TABLE "sentry_groupasignee" ADD CONSTRAINT "team_id_refs_id_5b32ca44" FOREIGN KEY ("team_id") REFERENCES "sentry_team" ("id") DEFERRABLE INITIALLY DEFERRED; []
   = CREATE INDEX "sentry_groupasignee_team_id" ON "sentry_groupasignee" ("team_id"); []
```

Reply:
lol